src/uct/api/v2/uct_v2.h

```diff
@@ -671,7 +671,10 @@ typedef enum uct_md_attr_field {
     UCT_MD_ATTR_FIELD_EXPORTED_MKEY_PACKED_SIZE = UCS_BIT(13),
 
     /** Unique global identifier of the memory domain. */
-    UCT_MD_ATTR_FIELD_GLOBAL_ID                 = UCS_BIT(14)
+    UCT_MD_ATTR_FIELD_GLOBAL_ID                 = UCS_BIT(14),
+
+    /** Indicate memory types that are suitable for non-blocking registration */
```

Comment:
`Bitmap of memory types that support @ref UCT_MD_MEM_FLAG_NONBLOCK flag for @ref uct_md_mem_reg or @ref uct_md_mem_reg_v2`